src/common/com/intellij/plugins/haxe/ide/annotator/HaxeSemanticAnnotator.java

```diff
@@ -96,8 +104,10 @@ public void run() {
       });
     }
     else if (requireConstant && type2.getType().getConstant() == null) {
+      //if(!HaxeAbstractEnumUtil.checkExpressionIsConst(initExpression.getExpression())) {
```

Comment:
Is there a reason to leave these lines commented?  Obviously, they were testing code, but they no longer appear to have any function.